addons/mail/static/src/new/rtc/rtc_service.js

```diff
@@ -116,6 +116,9 @@ export class Rtc {
             this.toggleVideo("camera");
             void proxyBlur.useBlur;
         }).useBlur;
+        this.ringingThreads = reactive([], () => this.onRingingThreadsChange());
+        this.ringingThreads.length;
+        this.store.ringingThreads = this.ringingThreads;
```

Comment:
Isn't it possible to just have `this.store.ringingThreads` in code rather than also `this.ringingThreads` ?